code/datums/gamemode/factions/legacy_cult/soulstone.dm

```diff
@@ -24,8 +24,8 @@
 /obj/item/device/soulstone/attack(var/mob/living/M, mob/user as mob)
 	if(!istype(M, /mob/living/carbon) && !istype(M, /mob/living/simple_animal))
 		return ..()
-	if(istype(M, /mob/living/carbon/human/manifested))
-		to_chat(user, "The soul stone shard seems unable to pull the soul out of that poor manifested ghost back onto our plane.")
+	if(ismanifested(M))
+		to_chat(user, "\The [src] shard seems unable to pull the soul out of that powerful body.")
```

Comment:
shouldn't it just be \the [src]

Reply:
` An object expression will display the object's name preceded by the text macro \the or \The if no other article has been specified.`

Reply:
no, i mean, drop the shard after [src] unless you want a soul stone shard shard

Reply:
They might be ranting about the not-really-oop way of handling this

Reply:
OH